packages/docs/concepts/blocks.yaml

```diff
@@ -24,56 +24,35 @@ _ref:
+            - `blocks: array`: A array of blocks to render to the default `content` area for `container` and `list` blocks. See [layout](/layout) for more details on how to use the `blocks` array.
             - `events: object`: Used to defined [`actions`](/events-and-actions) that run when the block triggers an [`event`](/events-and-actions).
             - `layout: object`: Used to define the [layout](/layout) properties for a block. __Operators are evaluated__.
             - `skeleton: object`: Used to overwrite a block's default loading behavior by rendering a skeleton. Any block types from [`@lowdefy/blocks-basic`](https://www.npmjs.com/package/@lowdefy/blocks-basic) and [`@lowdefy/blocks-loaders`](https://www.npmjs.com/package/@lowdefy/blocks-loaders) can be used in the skeleton definition, including sub-blocks.
             - `loading: boolean`: When true, renders the block in loading. If a `skeleton` is defined, it will be rendered, else the block's default loading behavior will be rendered.
-            - `required: boolean | string`: For `input` blocks, whether or not a value value is required in `state` when the [`Validate`](/Validate) action is called. Can be either a boolean or a string that is used as the validation error message . __Operators are evaluated__.
+            - `required: boolean | string`: For `input` blocks, whether or not a value is required in `state` when the [`Validate`](/Validate) action is called. Can be either a boolean or a string that is used as the validation error message . __Operators are evaluated__.
             - `style: css object`: Used to apply css style settings to the block's top level `div` element. __Operators are evaluated__.
             - `validate: array`: A list of validation tests to pass when the [`Validate`](/Validate) action is called. __Operators are evaluated__.
             - `visible: boolean`: Controls whether or not to render a block. Operators  are generally used here, and must evaluate to `false` to make the block invisible. Blocks with `visible: false` are excluded from `state`. __Operators are evaluated__.
 
-            # Block types
+            ## Block types
 
-            Lowdefy has a list of default block types as defined in the Lowdefy docs. The default Lowdefy blocks aim to cover a very generic implementation of the [Ant Design](https://ant.design/components/overview/) react component library. To use all the default block types, you can simply use the block `type` key, like [`Button`](/Button), [`TextInput`](/TextInput), or [`Box`](/Box).
+            Lowdefy has a list of default block types as defined in the Lowdefy docs. The default Lowdefy blocks aim to cover a very generic implementation of the [Ant Design](https://ant.design/components/overview/) react component library. To use all the default block types, you can simply use the block `type` key, such as [`Button`](/Button), [`TextInput`](/TextInput), or [`Box`](/Box).
```

Comment:
@Gervwyk we do a lot more blocks than just Ant Design - not sure if we want to say this still or rephrase it a bit

Reply:
Maybe this:

> Most of the default Lowdefy blocks aim to cover a very generic implementation of the [Ant Design](https://ant.design/components/overview/) react component library, but are not limited to Ant Design since blocks are just simple React components.

Reply:
Change made in b4bcd68767166479063cf5cf27a7b242846eae2c